factorial_02.py

```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
+    def factorial(num): #Using Recursive Method
+        if num == 0 or num ==1:
+            return 1
+        else:
+            return num * factorial(num-1)
+    print(f"Factorial of {num} is {factorial(num)}") #Recursive Method Answer
+except:
```

Comment:
Please explain about except in multiline comments at the end of program

Reply:
#The except block lets you handle the error.